src/main/java/com/khoros/twitterapp/services/TwitterService.java

```diff
@@ -206,7 +237,7 @@ private void handleTwitterException(TwitterException e) throws TwitterServiceExc
 
         }
 
-        throw new TwitterServiceException("Twitter Exception thrown.", e);
+        return new TwitterServiceException("Twitter Exception thrown.", e);
```

Comment:
why is this returning?

Reply:
I'm returning it to the original method throwing it there